docroot/themes/custom/uids_base/scss/global.scss

```diff
@@ -741,3 +741,16 @@ body:not(.page-node-type-article, .page-node-type-page).layout-builder-disabled
 .screen-reader-text {
   @include element-invisible;
 }
+
+.block-inline-blockuiowa-image {
+  img {
+    width: 100%;
+  }
+  a:focus-visible img {
+    outline: 1px solid blue;
+  }
+  a:focus-visible img,
+  a:hover img {
+    opacity: .6;
+  }
+}
```

Comment:
Instead of this going here, there should be a library added for the image block in `uids_base.libraries.yml`. A separate SCSS file should be created for the image block, following how we handle adding CSS for other block types.

Reply:
In a huddle with @bspeare, we thought it was light enough at the time to just include it in global.scss and if it was developed further we would move it out. That was when it was just `width: 100%` so I wonder if that line has been crossed

Reply:
If we are not going to create a library for this block here, then I think a follow-up task needs to be created to document that it needs to be done. Otherwise, in the future when people are looking for how to change stuff related to the block, it is not going to be obvious where it needs to be done. Having individual libraries per block makes it clear where future updates need to go.

Reply:
I can move this to uids_base, if someone can help guide me a bit.

Reply:
This was moved to a separate library